ci/README.license

```diff
@@ -0,0 +1,2 @@
+SPDX-FileCopyrightText: Red Hat, Inc.
+SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
Why not in the README file?

Reply:
Same as for the `vdsm_hooks` READMEs, as this is just plain text file, it has no syntax for comments and copyright headers would be visible in the project README.

Reply:
Including in the text of plain text readme is fine. If the maintainer of the plain text
readme does not like this they can convert the readme to markdown. Adding many .license files makes it harder to maintain the code.